doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
AMDG

On 04/29/2016 01:18 PM, Edward Diener wrote:
>> @@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
>>        <varlistentry>
>>          <term><literal>cflags</literal></term>
>>          <term><literal>cxxflags</literal></term>
>> +        <term><literal>compileflags</literal></term>
> 
> I do not understand how 'cflags' can mean command line options for both C and C++ when the various compiler toolsets clearly state that 'cflags' specifies command line options for C mode and that 'compileflags' specifies command line options for both C and C++ mode. Can you explain this to me ?
> 

  You're confusing cflags as an option for
toolset initialization with cflags as a feature.
The arguments for 'using xxx' are toolset specific
and are essentially named parameters.  They are
completely independent from the feature/property
framework, even though they share the same syntax.

In Christ,
Steven Watanabe